### PR TITLE
Completed CRUD actions for Register Apps

### PR DESCRIPTION
**Pull request by praveenyr**
**Description:** Completed CRUD actions for _Register Apps_  user story.

**Test Coverage:** 95.79%

### CODE REVIEW

File: app/assets/javascripts/registered_applications.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (brittanyjspringer) [inline app/assets/javascripts/registered_applications.coffee]: @praveenyr this is not being used so please remove.

File: app/assets/stylesheets/registered_applications.scss [hunk 2]
```diff
@@ -0,0 +1,7 @@
+// Place all the styles related to the RegisteredApplications controller here.
```
Comment (brittanyjspringer) [inline app/assets/stylesheets/registered_applications.scss]: @praveenyr this is not being used so please remove.

File: coverage/.last_run.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (brittanyjspringer) [inline coverage/.last_run.json]: @praveenyr please add /coverage/* to your .gitignore file and delete the coverage folder.

File: spec/helpers/registered_applications_helper_spec.rb [hunk 4]
```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
```
Comment (brittanyjspringer) [inline spec/helpers/registered_applications_helper_spec.rb]: @praveenyr this is not being used so please remove.

**Review by brittanyjspringer — CHANGES_REQUESTED** [review]

**Comment by praveenyr** [comment]
@wonderwoman13  - all set.

**Review by brittanyjspringer — APPROVED** [review]

**Pull request merged**